app/src/main/java/com/pvp/app/ui/router/Routes.kt

```diff
@@ -78,15 +80,15 @@ sealed class Route(
         iconDescription = "Calendar page button icon",
         path = "calendar",
         resourceTitleId = R.string.route_calendar,
-        screen = { _, _ -> CalendarScreen() }
+        screen = { _, _ -> VerticalPagerScreen(currentPage = 1) }
     )
 
     data object MonthlyCalendar : Route(
```

Comment:
This should be dropped, since Calendar screen is capable of navigating to Monthly calendar and back again. So just `Calendar` route is enough.

I would also suggest renaming `CalendarScreen` file and component into `CalendarWeeklyScreen`, `MonthlyCalendarScreen` into `CalendarMonthlyScreen` and `VerticalPager` into `CalendarScreen`, which would resolve which page currently is being shown.